sdm/src/test/java/integration/com/sap/cds/sdm/AttachmentsSDMTest.java

```diff
@@ -0,0 +1,473 @@
+  private static String entityID;
+  private static String entityID2;
+  private static String appUrl;
+  private static String authUrl;
+  private static String username;
+  private static String password;
+  private static String serviceName = "AdminService";
+  private static String entityName = "Books";
+  private static String srvpath = "AdminService";
+  private static Api api;
+  private static String attachmentID1 = "";
+  private static String attachmentID2 = "";
+  private static String attachmentID3 = "";
+  private static String attachmentID4 = "";
+
+  @BeforeAll
+  public static void setup() throws IOException {
+    // Define your clientId and clientSecret
+    Properties credentialsProperties = Credentials.getCredentials();
+    String clientId = credentialsProperties.getProperty("clientID");
+    String clientSecret = credentialsProperties.getProperty("clientSecret");
+    appUrl = credentialsProperties.getProperty("appUrl");
+    authUrl = credentialsProperties.getProperty("authUrl");
+    username = credentialsProperties.getProperty("username");
+    password = credentialsProperties.getProperty("password");
+
+    // Encode clientId:clientSecret to Base64
+    String credentials = clientId + ":" + clientSecret;
+    String basicAuth =
+        "Basic " + Base64.getEncoder().encodeToString(credentials.getBytes(StandardCharsets.UTF_8));
+
+    OkHttpClient client = new OkHttpClient().newBuilder().build();
+    MediaType mediaType = MediaType.parse("text/plain");
+    RequestBody body = RequestBody.create(mediaType, "");
+    Request request =
+        new Request.Builder()
+            .url(
+                authUrl
+                    + "/oauth/token?grant_type=password&username="
+                    + username
+                    + "&password="
+                    + password)
+            .method("POST", body)
+            .addHeader("Authorization", basicAuth)
+            .build();
+    Response response = client.newCall(request).execute();
+    token = new ObjectMapper().readTree(response.body().string()).get("access_token").asText();
+    response.close();
+    Map<String, String> config = new HashMap<>();
+    config.put("Authorization", "Bearer " + token);
+    api = new Api(config);
+  }
+
+  @Test
+  @Order(1)
+  public void testCreateEntityAndCheck() throws IOException {
+    System.out.println("Test (1) : Create entity and check if it exists");
+    Boolean testStatus = false;
+    String response = api.createEntityDraft(appUrl, serviceName, entityName, srvpath);
+    if (response != "Could not create entity") {
+      entityID = response;
+      response = api.saveEntityDraft(appUrl, serviceName, entityName, srvpath, entityID);
+      if (response == "Saved") {
+        response = api.checkEntity(appUrl, serviceName, entityName, entityID);
+        if (response.equals("Entity exists")) {
+          testStatus = true;
+        }
+      }
+    }
+    if (!testStatus) {
+      fail("Could not create entity");
```

Comment:
Please add the actual error message instead of "Could not create entity"

Reply:
Print statements are added in api.java